### DISCUSSION TITLE
Is it possible to update schema once created?

### DISCUSSION THREAD

**Discussion by perfecto25** [fastapi/sqlmodel#1498] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
class Xconnect(SQLModel, table=True):
    id: int = Field(primary_key=True)
    lvc: Optional[str] = Field(sa_column_kwargs={"unique": True})
    customer: int = Field(foreign_key="customer.id")
    region: int = Field(foreign_key="attr.id")
```

### Description

Im trying to see if I can update the exisitng generated table Xconnect without deleting and recreating it,

for example, if I wantto make the Region field Optional, adding

```
region: Optional[int]
```

will not update the existing schema bcs the table is already created. How can I update the existing schema? Do I need to use Alembic for this?

### Operating System

Linux

### Operating System Details

linux mint 19

### SQLModel Version

0.0.5

### Python Version

3.8

**Comment by voxofox** [comment, marked as answer]
As you've mentioned, Alembic would be a good solve.
